zingolib/src/commands.rs

```diff
@@ -732,8 +732,20 @@ impl Command for AddressCommand {
         "List all addresses in the wallet"
     }
 
-    fn exec(&self, _args: &[&str], lightclient: &LightClient) -> String {
-        RT.block_on(async move { lightclient.do_addresses().await.pretty(2) })
+    fn exec(&self, args: &[&str], lightclient: &LightClient) -> String {
+        match args.len() {
+            0 => self.help().to_string(),
+            1 => match args[0] {
+                "true" => {
+                    RT.block_on(async move { lightclient.do_addresses(true).await.pretty(2) })
+                }
+                "false" => {
+                    RT.block_on(async move { lightclient.do_addresses(false).await.pretty(2) })
+                }
+                _ => self.help().to_string(),
+            },
+            _ => self.help().to_string(),
+        }
```

Comment:
Maybe can be better if `args.len === 0` the command assume `false` (it would work as before... no breaking changes...) IMO.

Reply:
I changed it, so that you *must* say "truie" or "false"..  which is a little bit more work, but forces the reader to understand more.